geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -354,20 +357,31 @@ class SelectionMapFragment(
         map.clearFeatures()
         itemsByFeatureId.clear()
 
-        val markerDescriptions = items.map {
+        val singlePoints = items.filter { it.points.size == 1 }
+        val traces = items.filter { it.points.size != 1 }
+
+        val markerDescriptions = singlePoints.map {
+            val point = it.points[0]
+
             MarkerDescription(
-                MapPoint(it.latitude, it.longitude),
+                MapPoint(point.latitude, point.longitude),
                 false,
                 MapFragment.BOTTOM,
                 MarkerIconDescription(it.smallIcon, it.color, it.symbol)
             )
         }
 
-        val featureIds = map.addMarkers(markerDescriptions)
-        items.zip(featureIds).forEach { (item, featureId) ->
+        val pointIds = map.addMarkers(markerDescriptions)
+        val traceIds = traces.fold(listOf<Int>()) { ids, item ->
+            ids + map.addDraggablePoly(item.points, false)
```

Comment:
Those traces are draggable what seems to be weird and I think they shouldn't be.

Reply:
Oh, wow! I'd thought this was just a badly named method, but you totally can drag them (I hadn't been able to drag them as it's a little awkward on the emulator before, but eventually managed it). Will fix that.